cognite/client/_api/assets.py

```diff
@@ -171,12 +175,21 @@ def __call__(
             external_id_prefix=external_id_prefix,
         ).dump(camel_case=True)
 
+        prep_sort = None
+        if sort is not None:
+            prep_sort = prepare_filter_sort(sort, AssetSort)
```

Comment:
This check is not needed as it is handled by the function `prepare_filter_sort`

Reply:
updated